twitch.go

```diff
@@ -178,6 +181,24 @@ func scrolloReward(params string) {
 	f.WriteString(fmt.Sprintf(" %.256s ✨✨✨ ", params))
 }
 
+func playYoutubeReward(params string) {
+	var isValidID = regexp.MustCompile(`[a-zA-Z0-9_-]`).MatchString
+	url_args := strings.Split(params, "v=")[1] // Get the video ID arg from the rest of the URL
+	video_id := strings.Split(url_args, "&")[0] // Split out any other args from the URL, keeping only the video ID
+
+	if !(len(video_id)==11) {
+		log.Printf("video ID too long/short:",len(video_id))
+		return
+	}
+	if (!isValidID(video_id)) {
+		log.Printf("malformed video ID")
+		return
+	}
+
+	cmd := exec.Command("./play_yt.sh", video_id)
+	cmd.Run()
```

Comment:
we might need to `go cmd.Run()` so that we don't block finishing up handling this request whilst we're playing the thing

Reply:
Every time I tried `go cmd.Run()` it didn't work. I'm happy to include that in the changes to see if it's a machine-specific issue!

Reply:
/shruggie

nbd then i guess